src/account/AccountInfoQuery.ts

```diff
@@ -43,6 +44,15 @@ export class AccountInfoQuery extends QueryBuilder<AccountInfo> {
         return this;
     }
 
+    public async getCost(client: BaseClient): Promise<Hbar> {
+        // deleted accounts return a COST_ANSWER of zero which triggers `INSUFFICIENT_TX_FEE`
+        // if you set that as the query payment; 25 tinybar seems to be the minimum to get
```

Comment:
It's not necessarily the "minimum" for `AccountInfoQuery`, it's just probably *enough*.